packages/reactive-element/src/reactive-element.ts

```diff
@@ -553,7 +554,7 @@ export abstract class ReactiveElement
    * @nocollapse
    * @category properties
    */
-  static elementProperties: PropertyDeclarationMap = new Map();
+  static [elementProperties]: PropertyDeclarationMap = new Map();
```

Comment:
Why is this suddenly needed, when this property has existed since the first lit-element version? There are other static properties that don't have this need... what's different about them?